pkg/log/log.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import (
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"gopkg.in/natefinch/lumberjack.v2"
+)
+
+// Logger wraps logrus.Entry
+type Logger struct {
+	*logrus.Entry
+}
+
+// SetOutput sets log output to filename globally.
+// filename="/var/log/sqlflow.log": write the log to file
+// filename="": write the file to stdout
+func SetOutput(filename string) {
```

Comment:
Do we need to add this source file?

Is seems that the only place we need to call `log.SetOutput` is in `main`. We can define `log.SetOutput` function in the source file where `main` resides and names it `setLogrusOutput`.

Also, we don't need to add the type `Logger` in addition to `logurs.Entry`, nor do we need `log.WithFields`.

Reply:
@wangkuiyi Following your suggestion, do we still need this `log` package?